IFC4x3/Templates/Project Context/Project Global Positioning Mapped/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Project_Global_Positioning_Mapped_1N2lpctor8Fhouaa7TR5Hv" Name="Project Global Positioning Mapped" UniqueId="570afce6-df2d-483e-bcb8-9241dd6c5479" Type="IfcContext">
+	<Rules>
+		<DocModelRuleAttribute Name="RepresentationContexts">
+			<Rules>
+				<DocModelRuleEntity Name="IfcGeometricRepresentationContext">
+					<Rules>
+						<DocModelRuleAttribute Name="HasCoordinateOperation" Identification="HasGlobalPosition">
+							<Rules>
+								<DocModelRuleEntity Name="IfcRigidOperation">
+									<Rules>
+										<DocModelRuleAttribute Name="TargetCRS">
+											<Rules>
+												<DocModelRuleEntity Name="IfcProjectedCRS">
+													<Rules>
+														<DocModelRuleAttribute Name="HasWellKnowTextRepresentation" Identification="WKTRepresentation">
+															<Rules>
+																<DocModelRuleEntity Name="IfcWellKnownText">
+																	<Rules>
+																		<DocModelRuleAttribute Name="WellKnownText" Identification="WKT">
```

Comment:
One thing I would suggest here is to create a partial for WKT. In order to be able to re-use it.